jekyll-github-metadata.gemspec

```diff
@@ -23,5 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "netrc"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
-  spec.add_development_dependency "rubocop", "0.51"
+  spec.add_development_dependency "rubocop", "~> 0.59.0"
```

Comment:
Should we be using our version of rubocop? Or for another PR?

Reply:
As Jekyll 3.8.3 still target 2.1, we can lock to 0.57 for now, I'll update this PR.